<launch>

  <!-- Joy node -->
  <node pkg="joy" type="joy_node" name="joy_node" output="screen">

    <param name="dev" value="/dev/input/js0" type="string "/>
    <param name="deadzone" value="0.05" type="double"/>    
    <param name="autorepeat_rate" value="1000" type="double"/> 

  </node>
 
  <node pkg="control_4ws4wd" type="chassis_4ws4wd_node" name="chassis_4ws4wd" output="screen">
  	 <param name="l_scale_1" value="800" type="double"/> 
  	 <param name="w_scale_1" value="400" type="double"/> 
  	 <param name="l_scale_2" value="1500" type="double"/> 
  	 <param name="w_scale_2" value="900" type="double"/> 
  	 
<!--      <remap from="/cmd_vel" to="/chassis_4ws4wd_node/cmd_vel"/>-->
  </node>

</launch>
